CMakeLists.txt

```diff
@@ -118,6 +118,12 @@ endif()
 
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules)
 
+# Always statically link libstdc++ and libgcc for portability
+if (NOT MSVC)
```

Comment:
Would this break compatability with clang if anyone is using libc++?

Reply:
https://stackoverflow.com/a/78242553/2904531

Reply:
Looks like there is a way to detech if `libstdc++` is in use.

Reply:
seems it works for clang as well. but I didn't verify it. so I add gnu check for now.